x-pack/plugins/cases/public/components/formatted_date/index.tsx

```diff
@@ -93,7 +93,7 @@ PreferenceFormattedP1DTDate.displayName = 'PreferenceFormattedP1DTDate';
  * - the raw date value (e.g. 2019-03-22T00:47:46Z)
  */
 export const FormattedDate = React.memo<{
-  fieldName: string;
+  fieldName?: string;
```

Comment:
I use this in the status metrics. This gets passed to `LocalizedDateTooltip` which accepts it as optional, so I'm just keeping it as optional. I don't believe this was used anywhere else in cases. I could have missed something though.